vim_onenote.ahk

```diff
@@ -196,41 +295,70 @@ Send, {ShiftUp}
 return
 
 ; Delete current line
+; dd handled specially, to delete newline.
 d::
-if IsLastKey("d")
-{
-    Send, {Home}{ShiftDown}{End}
-    Send, ^c ; Yank before delete, don't use cut so blank lines are deleted 
+    SelectMotion()
+    send ^x
+    if IsLastKey("d")
     Send, {Del}
-    Send, {Shift}
-}
-return 
+return
 
-+s::
++S::
 Send, {Home}{ShiftDown}{End}
 Send, ^x ; Cut instead of yank and delete
 Send, {ShiftUp}
 Gosub, InsertMode   
 return 
 
-return 
 ; TODO handle regular paste , vs paste something picked up with yy
```

Comment:
Looked over this function again. That isn't what it meant. Currently it pastes onto a new line, ie it assumes linewise yank/delete.
I'd implement this with a flag variable, but setting it and unsetting it would be a pain. Thing for the future.